pkg/armhelpers/graph.go

```diff
@@ -28,6 +29,11 @@ func (az *AzureClient) CreateGraphApplication(ctx context.Context, applicationCr
 	return az.applicationsClient.Create(ctx, applicationCreateParameters)
 }
 
+// DeleteGraphApplication creates an application via the graphrbac client
```

Comment:
This comment should say "deletes an application".

Reply:
yup ! I ll change it now